lib/cogctl/actions/roles/info.ex

```diff
@@ -16,22 +16,23 @@ defmodule Cogctl.Actions.Roles.Info do
   end
 
   defp do_info(endpoint, role_name) do
-    case CogApi.HTTP.Roles.show(endpoint, %{name: role_name}) do
+    case CogApi.HTTP.Client.role_show(endpoint, %{name: role_name}) do
       {:ok, role} ->
-        role_attrs = [[Map.fetch!(role, :name), Map.fetch!(role, :id)]]
+        role_attrs = [["NAME", "ID"]] ++ [[Map.fetch!(role, :name), Map.fetch!(role, :id)]]
 
-        permission_attrs = Enum.map(role.permissions, fn(permission) ->
+        permission_attrs = [["NAMESPACE", "NAME", "ID"]] ++ 
```

Comment:
We're doing this same kind of dance in a few commands, now, right? We should have some shared utility functions that generate these data instead of repeating.